packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -319,13 +320,17 @@ export default class extends Component {
       )
     }
 
+    const mdadmHealth = await getMdadmHealth(host).catch(console.error)
+
     this.setState({
       isHtEnabled: await isHyperThreadingEnabledHost(host).catch(() => null),
       isSmartctlHealthEnabled,
       pciStateById,
       smartctlUnhealthyDevices,
       unhealthyDevicesAlerts,
       isPciPassthroughAvailable: _isPciPassthroughAvailable,
+      mdadmHealthAvailable: mdadmHealth?.raid ? mdadmHealth : null,
```

Comment:
`mdadmHealthAvailable` seems to indicate that we are expecting a boolean.
```suggestion
      mdadmHealthAvailable: mdadmHealth != null ? true : false,
```